### PR TITLE
[Maps Search] Update names based on APIView feedback for Preview 1.0

### PR DESCRIPTION
**Pull request by alextts627**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/alankashiwa/azure-rest-api-specs/maps-geolocation-improvement-for-sdk...main/).

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [x] **To improve swagger for generated clients and address some issues from [this review in ApiView](https://apiview.dev/Assemblies/Review/0ab748be5297432eaf27d512e2d1b998)**
  2. When are you targeting to deploy the new service/feature to public regions? **Services already deployed.**
  3. When do you expect to publish the swagger? **ASAP**
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published. **No data-plane SDKs are onboarded yet, so no refresh is required.**

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further questions about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [x] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

### CODE REVIEW

File: specification/maps/data-plane/Search/preview/1.0/search.json [hunk 1]
```diff
@@ -453,7 +454,7 @@
           {
             "name": "geometries",
             "in": "query",
-            "x-ms-client-name": "geometryIds",
+            "x-ms-client-name": "listPolygons",
```
Comment (tjprescott) [inline specification/maps/data-plane/Search/preview/1.0/search.json:457]: Was this from the Python APIView or something else? This _seems_ like a strange name.

**Review by tjprescott — COMMENTED** [review]
One question, but otherwise looks good to me.

File: specification/maps/data-plane/Search/preview/1.0/search.json (hunk 1 above)
Comment (alextts627) [reply specification/maps/data-plane/Search/preview/1.0/search.json:457]: @tjprescott  I might misunderstood your comments on `async def get_polygon` method. I will change it back.

**Comment by alextts627** [comment]
Hi @tjprescott This PR was mean to be the updated of the swagger following by the feedback you gave, mostly are name changes for the parameters.

For rest of the issues such as Coordinate type, optional params, etc.. I believe those will be done in the convenience layer for the SDK.

**Review by tjprescott — APPROVED** [review]

**Pull request merged**